src/onboarding/welcome/Welcome.tsx

```diff
@@ -87,49 +88,48 @@ export default function Welcome() {
 
   return (
     <SafeAreaView style={styles.container}>
-      <ScrollView contentContainerStyle={styles.contentContainer}>
```

Comment:
Should this scroll view be retained?

Reply:
I think the scroll view was there for the welcome text that was being shown, now it is just the valora icon so not sure if it makes sense to have that be in a scroll view since it should just be in the center of the screen and not change size based on text size on the phone